model/wd2/index-nametemplate.html

```diff
@@ -385,13 +385,19 @@
 <br/>Bodies or Targets which are External Web Resources MUST have exactly 1 <code>id</code> with the value of the resource's <a>IRI</a>.
   </td></tr>
   <tr><td>language</td><td>Property</td><td>The language of the Web Resource's content
-  <br/>The Body or Target SHOULD have exactly 1 language associated with it, but MAY have 0 or more.  The value of the property SHOULD be a language code following the [[!bcp47]] specifiction.</td></tr>
+  <br/>The Body or Target SHOULD have exactly 1 language associated with it, but MAY have 0 or more.  The value of the property SHOULD be a language code following the [[!bcp47]] specification.</td></tr>
 <tr><td>format</td><td>Property</td><td>The format of the Web Resource's content
   <br/>The Body or Target SHOULD have exactly 1 format associated with it, but MAY have 0 or more.  The value of the property SHOULD be the media-type of the format, following the [[!rfc6838]] specification.</td></tr>
+<tr><td>textDirection</td><td>Relationship</td><td>The direction of the text of the resource.  
+<br/>The Body or Target MAY have exactly 1 textDirection associated with it.  The value of the property MUST be one of the directions defined below (<code>ltr</code>, <code>rtl</code>, or <code>auto</code>).
+</td></tr>
+<tr><td>ltr</td><td>Instance</td><td>The direction that indicates the value of the resource is explicitly directionally isolated left-to-right text.</td></tr>
+<tr><td>rtl</td><td>Instance</td><td>The direction that indicates the value of the resource is explicitly directionally isolated right-to-left text.</td></tr>
+<tr><td>auto</td><td>Instance</td><td>The direction that indicates the value of the resource is explicitly directionally isolated text, and the direction is to be programmatically determined using the value.</td></tr>
 </table>
 
 <div class="note">
-The [[iana-media-types]] document provides the official registry of media types that can be used with the <code>format</code> property.  The [[w3c-language-tags]] article provides a good overview of the values that implementers can expect to encounter in the <code>language</code> property.
+The [[iana-media-types]] document provides the official registry of media types that can be used with the <code>format</code> property.  The [[w3c-language-tags]] article provides a good overview of the values that implementers can expect to encounter in the <code>language</code> property.  The notion of text direction is taken explicity from <a href="https://www.w3.org/TR/html5/dom.html#the-dir-attribute">HTML5's</a> <code>dir</code> attribute.
```

Comment:
The reference is tricky, because the HTML folks have specific policy about the short names. Indeed:
* /TR/html is the reference to the latest HTML version (currently HTML5
* /TR/html5 is the current HTML5
* /TR/html5.1 is the upcoming HTML5.1 (which will obsolete HTML5), etc.

Ie, this reference might become irrelevant in future as being updated? It may be safer to either refer to HTML (using a [[...]] respec trick) and simply refer to the attribute name in HTML, or use /TR/html, trusting that the URL will remain valid...